zerver/tests/test_events.py

```diff
@@ -923,6 +925,7 @@ def test_revoke_user_invite_event(self) -> None:
         events = self.verify_action(
             lambda: do_revoke_user_invite(prereg_users[0]),
             state_change_expected=False,
+            num_events=2,
```

Comment:
I think duplicate events are being sent here for admins here.

Reply:
Yes, earlier there were duplicate eevents. I have made changes now.